docs/wiki/rocksdb-in-cita.md

```diff
@@ -0,0 +1,89 @@
+## RocksDB 简单介绍
+当前 CITA 使用 RocksDB 作为其底层存储的数据库。
+RocksBD 采用的是 LSM-Tree(Log-Structured-Merge-Tree), 如下图所示：
+
+<img src = "https://github.com/cryptape/cita/blob/develop/docs/wiki/images/rocksdb-lsm-tree.png?raw=true" width="50%" height="50%">
```

Comment:
Does `![LSM-Tree](images/rocksdb-lsm-trr.png)`  work?

Reply:
I have tried some relative links in citahub-doc, and it doesn't work.

Reply:
Not citahub-doc, just check if github can show the pic.